lib/fog/openstack/core.rb

```diff
@@ -198,7 +198,7 @@ def setup(options)
 
         @openstack_service_type = options[:openstack_service_type] || default_service_type
         @openstack_endpoint_type = options[:openstack_endpoint_type] || default_endpoint_type
```

Comment:
This still looks redundant, we should also drop the duplicate assignment in L177.

Reply:
I was wondering the same thing while looking into the code. Shall I go ahead and remove it?